rosetta-source/src/main/resources/cdm-sample-files/fpml-5-13/products/fx-derivatives/expectations.json

```diff
@@ -1,18 +1,18 @@
-{"fileName":"cdm-sample-files/fpml-5-13/products/fx-derivatives/fx-ex31-volatility-swap.xml","excludedPaths":-1,"externalPaths":55,"outstandingMappings":12,"validationFailures":15,"qualificationExpectation":{"success":true,"qualifyResults":[{"qualifiedName":"ForeignExchange_ParameterReturnVolatility","qualifiedObjectClass":"cdm.product.template.EconomicTerms"}],"qualifiableObjectCount":1}}]
\ No newline at end of file
+{"fileName":"cdm-sample-files/fpml-5-13/products/fx-derivatives/fx-ex01-fx-spot.xml","excludedPaths":-1,"externalPaths":37,"outstandingMappings":15,"validationFailures":2,"qualificationExpectation":{"success":true,"qualifyResults":[{"qualifiedName":"ForeignExchange_Spot_Forward","qualifiedObjectClass":"cdm.product.template.EconomicTerms"}],"qualifiableObjectCount":1}},
+{"fileName":"cdm-sample-files/fpml-5-13/products/fx-derivatives/fx-ex02-spot-cross-w-side-rates.xml","excludedPaths":-1,"externalPaths":45,"outstandingMappings":23,"validationFailures":2,"qualificationExpectation":{"success":true,"qualifyResults":[{"qualifiedName":"ForeignExchange_Spot_Forward","qualifiedObjectClass":"cdm.product.template.EconomicTerms"}],"qualifiableObjectCount":1}},
+{"fileName":"cdm-sample-files/fpml-5-13/products/fx-derivatives/fx-ex03-fx-fwd.xml","excludedPaths":-1,"externalPaths":39,"outstandingMappings":15,"validationFailures":2,"qualificationExpectation":{"success":true,"qualifyResults":[{"qualifiedName":"ForeignExchange_Spot_Forward","qualifiedObjectClass":"cdm.product.template.EconomicTerms"}],"qualifiableObjectCount":1}},
+{"fileName":"cdm-sample-files/fpml-5-13/products/fx-derivatives/fx-ex05-fx-fwd-w-ssi.xml","excludedPaths":-1,"externalPaths":42,"outstandingMappings":18,"validationFailures":2,"qualificationExpectation":{"success":true,"qualifyResults":[{"qualifiedName":"ForeignExchange_Spot_Forward","qualifiedObjectClass":"cdm.product.template.EconomicTerms"}],"qualifiableObjectCount":1}},
+{"fileName":"cdm-sample-files/fpml-5-13/products/fx-derivatives/fx-ex07-non-deliverable-forward.xml","excludedPaths":-1,"externalPaths":48,"outstandingMappings":15,"validationFailures":3,"qualificationExpectation":{"success":true,"qualifyResults":[{"qualifiedName":"ForeignExchange_NDF","qualifiedObjectClass":"cdm.product.template.EconomicTerms"}],"qualifiableObjectCount":1}},
+{"fileName":"cdm-sample-files/fpml-5-13/products/fx-derivatives/fx-ex08-fx-swap.xml","excludedPaths":-1,"externalPaths":52,"outstandingMappings":17,"validationFailures":3,"qualificationExpectation":{"success":true,"qualifyResults":[{"qualifiedName":"ForeignExchange_Swap","qualifiedObjectClass":"cdm.product.template.EconomicTerms"}],"qualifiableObjectCount":1}},
```

Comment:
outstanding mappings up by 4 because the payer and receiver of the non-base exchangedCurrency is no longer mapped. This applies to both nearLeg and farLeg